NOTES.WIN

```diff
@@ -35,17 +35,17 @@
  In addition to the requirements and instructions listed in INSTALL,
  these are required as well:
 
- - You need Perl.  We recommend ActiveState Perl, available from
+ - Perl. We recommend ActiveState Perl, available from
    https://www.activestate.com/ActivePerl. Another viable alternative
    appears to be Strawberry Perl, http://strawberryperl.com.
    You also need the perl module Text::Template, available on CPAN.
    Please read NOTES.PERL for more information.
 
- - You need a Microsoft Visual C compiler. There is ambiguity in which
-   versions are supported. It's probably safe to say that latest along
-   with couple of previous are fully supported. On the other hand we
-   know for sure that eldest one does not work. Everything between falls
-   to best-effort category.
+ - Microsoft Visual C compiler. Since we can't test them all, there is
+   natural uncertainty in which versions are supported. It's probably
```

Comment:
s/in which/what/

Reply:
Also, I'm uncertain about "there is natural uncertainty"...  let me ponder, it's currently late at night.

Reply:
I can't agree with "s/in which/what". "What" would work in "what is supported", but we are talking about a subset of multiple versions, and you can't say "what one". I've also rephrased "natural uncertainty in" as "unavoidable uncertainty about". Does it sound better? What I want to achieve is to say that we effectively can't answer overly specific questions about compiler versions.